src/actions/favorite.js

```diff
@@ -0,0 +1,12 @@
+export const TOGGLE_FAVORITE = 'TOGGLE_FAVORITE';
+export const UPDATE_FILTERED_FAVORITES = 'UPDATE_FILTERED_FAVORITES';
+
+export const toggleFavorite = (gif) => ({
+  type: TOGGLE_FAVORITE,
+  payload: gif
```

Comment:
As a note, it is a good practice to wrap the payload inside an object (this convention was first created by the `Flux` team, and it's recommend to use also in `Redux` for consistency).

```js
payload: { gif },
```